cloudsync/tests/test_provider.py

```diff
@@ -979,7 +1011,7 @@ def test_event_del_create(provider):
     provider.delete(info1.oid)
     info2 = provider.create(dest, temp2)
     infox = provider.create(dest2, temp2)
-    time.sleep(1)
+    time.sleep(2)
```

Comment:
Consider that, if you are extending the sleep time, then you are really waiting for some condition that you may be able to detect directly, and replace the sleep with a check for that condition instead? That way you won't wait more time than necessary when the condition completes quickly, and you won't flake on tests where it slips past two seconds...

Reply:
The issue here is flakiness for the filesystem provider running on windows -- we're waiting on events to come in, and they are generated by a 3rd party lib (watchdog) which appears to be slower on windows than other platforms (I think @reed-atakama looked into this recently for another project)

There is probably a way to wait for the event(s) more directly, but then in the failure case (event never comes) we would wait until the job times out.

On my machine sleep values less than 1 (0.9, 0.99) fail consistently, 1 is about 50/50, greater than 1 succeeds consistently. I set it to 2 to be safe / account for slower machines and any other factors that play into this behavior. Seems safer than the alternative.